### DISCUSSION TITLE
Is it possible to add a command to open the Windows Terminal App?

### DISCUSSION THREAD

**Discussion by ingomohr** [anb0s/EasyShell#190] [Q&A]
I recently switched from CMD to the [Windows Terminal App](https://www.microsoft.com/en-us/p/windows-terminal/9n0dx20hk701?activetab=pivot:overviewtab) to have one place for all terminals (including WSL).

I would like to configure EasyShell to provide a context menu entry to open the selected folder or project in the Windows Terminal App.
Yet I'm failing to do so.

Is there a way to configure this?

**Comment by anb0s** [comment]
I've found this:
https://docs.microsoft.com/de-de/windows/terminal/command-line-arguments?tabs=windows

**Comment by anb0s** [comment]
Open EasyShell preferences go to `(2) Command` and add new command:
![image](https://user-images.githubusercontent.com/95811/112133452-047fb680-8bcc-11eb-8d31-a41e0a1ffdb1.png)

Category `Open`
Name: `Windows Terminal`
Command: `wt -d ${easyshell:container_loc}`
Press `OK `and then `Apply and Close` (because of reloading command issues => will fix this later)

Reopen EasyShell preferences go to `(1) Menu` and create new menu and select new command:
![image](https://user-images.githubusercontent.com/95811/112134165-b919d800-8bcc-11eb-8fa0-4434a5b032e5.png)
Filter: `terminal`
Press `OK ` and then move the new menu entry with `Up` and `Down` to wanted place:
![image](https://user-images.githubusercontent.com/95811/112134334-ecf4fd80-8bcc-11eb-8fe9-37f43c712cce.png)
then `Apply and Close`

Now it should work:
![image](https://user-images.githubusercontent.com/95811/112134839-6d1b6300-8bcd-11eb-836e-22e139805d98.png)

![image](https://user-images.githubusercontent.com/95811/112134954-8f14e580-8bcd-11eb-821e-dcc2c51348bc.png)

**Reply by anb0s** [reply]
The command with title like done in other presets can be:
`wt -d "${easyshell:container_loc}" --title "${easyshell:project_name}"`

**Reply by anb0s** [reply]
The full and verified command is: `wt new-tab -d "${easyshell:container_loc}" --title "${easyshell:project_name}"`
See this [issue](https://github.com/microsoft/terminal/issues/9585)

**Reply by anb0s** [reply]
Add to Wiki: https://github.com/anb0s/EasyShell/wiki/Windows-Terminal-App

**Comment by anb0s** [comment]
Please try this and if you want create a new [issue](https://github.com/anb0s/EasyShell/issues) for addting this as preset to next v2.2 release ;)

**Reply by ingomohr** [reply]
Thank you @anb0s . Works like a charm!
Thanks for the Wiki-entry you've added, too. I'll share it with my team so that the others can use it too.

**Reply by anb0s** [reply]
You are welcome. I've created an [issue](https://github.com/anb0s/EasyShell/issues/191) to have presets and it will be included in next release.

**Reply by anb0s** [reply]
you can use new testing build now #192 to have the presets
